src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -609,11 +609,20 @@ void quorum_cop::process_quorums(cryptonote::block const& block) {
                         auto quorum = m_core.service_node_list.get_quorum(
                                 quorum_type::checkpointing, m_last_checkpointed_height);
                         if (!quorum) {
-                            // TODO(oxen): Fatal error
-                            log::error(
-                                    logcat,
-                                    "Checkpoint quorum for height: {} was not cached in daemon!",
-                                    m_last_checkpointed_height);
+                            if (height - m_last_checkpointed_height > 40)
```

Comment:
Two suggestions here:
1) Rewrite as `height > m_last_checkpointed_height + 40`, to avoid the possibility of overflow.  (I'm not sure that this could practically overflow, or what the consequences would be, but with the conversion to addition you just don't need to worry about whether there are preconditions that prevent it).
2) Rather than a hardcoded 40, perhaps we should use something like `VOTE_LIFETIME / 2` (I'm not 100% sure that is the right variable).  It's still fairly arbitrary, but it feels slightly better to me than a hard coded constant.

Reply:
Done